Readability.js

```diff
@@ -1678,15 +1678,18 @@ Readability.prototype = {
    *
    * @return boolean Whether or not we suspect parse() will suceeed at returning an article object.
    */
-  isProbablyReaderable: function() {
-    var nodes = this._doc.getElementsByTagName("p");
-    if (nodes.length < 5) {
-      return false;
-    }
-
-    var possibleParagraphs = 0;
+  isProbablyReaderable: function(helperIsVisible) {
+    var nodes;
+    if (typeof this._doc.querySelectorAll != "undefined")
```

Comment:
The idea would be to rely on a root node and an array of tag names passed as args instead of NodeList directly. Maybe that wouldn't be the most beautiful API on earth though:

```js
_concatNodeList: function(node, tagNames) {
  var nodes, doc = this._doc;
  if (typeof doc.querySelectorAll != "undefined") {
    nodes = doc.querySelectorAll(tagNames.join(","));
  } else {
    nodes = this._concatNodeLists.apply(this, tagNames.map(function(tag) {
      return doc.getElementsByTagName(tag);
    }));
  }
  // …
}
```